platform/src/main/resources/example.env

```diff
@@ -6,15 +6,17 @@
 ####################################################################################################
 
 # REQUIRED - The secret key of JWT encryption
-JWT_SECRET=secret
+secret.sep10.jwt_secret=secret
 
 # REQUIRED - The private key of the SEP-10 challenge.
 # We highly recommend that this private key should not be used to sign any transactions to submit to the Stellar
 # network.
-SEP10_SIGNING_SEED=SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
+secret.sep10.signing_seed=SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
 
 # REQUIRED - JWT secrets used to communicate between Anchor and Platform.
+SECRET.CALLBACK_API.AUTH_SECRET=myPlatformToAnchorSecret
 PLATFORM_TO_ANCHOR_SECRET=myPlatformToAnchorSecret
```

Comment:
Can these old secrets (PLATFORM_TO_ANCHOR_SECRET) be removed?

Reply:
My thought is that secrets would be fully reviewed on https://github.com/stellar/java-stellar-anchor-sdk/issues/570 and https://github.com/stellar/java-stellar-anchor-sdk/issues/576, but I think it's ok to remove these 2 now.

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/593/commits/78e10f8520ed35cb53edccfa0a20e30f1cb65c3a.